client/src/document/toolbar/edit-actions.tsx

```diff
@@ -88,6 +90,11 @@ export function EditActions({ folder }: { folder: string }) {
       <button className="delete" onClick={deleteDocument}>
         Delete document
       </button>
+      <Link
+        to={`/en-US/_create?initial_slug=${encodeURIComponent(params["*"])}`}
```

Comment:
...you can change this to:
```suggestion
        to={`/en-US/_create?initial_slug=${encodeURIComponent(slug)}`}
```